test/parallel/test-https-set-timeout-server.js

```diff
@@ -34,8 +35,8 @@ const tls = require('tls');
 const tests = [];
 
 const serverOptions = {
-  key: fs.readFileSync(`${common.fixturesDir}/keys/agent1-key.pem`),
-  cert: fs.readFileSync(`${common.fixturesDir}/keys/agent1-cert.pem`)
+  key: fs.readFileSync(`${fixtures.readKey('agent1-key.pem')}`),
```

Comment:
This line can simply be `key: fixtures.readKey('agent1-key.pem')`.

Reply:
👍